helper/keysutil/policy.go

```diff
@@ -959,12 +969,19 @@ func (p *Policy) Sign(ver int, context, input []byte, algorithm string) (*Signin
 	// Convert to base64
 	encoded := base64.StdEncoding.EncodeToString(sig)
 
-	template := p.VersionTemplate
-	if template == "" {
-		template = DefaultVersionTemplate
-	}
+	var prefix string
+	prefixRaw, ok := p.versionPrefixCache.Load(ver)
```

Comment:
Maybe factor this and the above code into a helper function that returns a string given the ver?